tools/metadata.json

```diff
@@ -1,7 +1,7 @@
 {
-  "StableReleaseTag": "v6.1.3",
+  "StableReleaseTag": "v6.2.0",
   "PreviewReleaseTag": "v6.2.0-rc.1",
   "ServicingReleaseTag": "v6.0.5",
-  "ReleaseTag": "v6.1.3",
+  "ReleaseTag": "v6.2.0",
   "NextReleaseTag": "v6.2.0-rc.1"
```

Comment:
We should make the CI System use NextReleaseTag so we can increment this and leave the preview alone in this case.

This is not block